pom.xml

```diff
@@ -71,7 +71,7 @@
     <dependency>
       <groupId>com.1stleg</groupId>
       <artifactId>jnativehook</artifactId>
-      <version>2.1.0</version>
+      <version>2.0.2</version>
```

Comment:
I would not like to downgrade the dependency.

Can you please validate the error also happens, if we would not create an instance of the jnativehook on linux? I think currently we create one on every OS but do not "register listeners" for linux based systems. Maybe not creating an instance already fixes the issue?

Reply:
if you  do not "register listeners" for linux it works on Java 8 +11  with the latest jnativehook version